crates/editor/src/element.rs

```diff
@@ -510,8 +511,9 @@ impl EditorElement {
 
         let multi_cursor_setting = EditorSettings::get_global(cx).multi_cursor_modifier;
         let multi_cursor_modifier = match multi_cursor_setting {
-            MultiCursorModifier::Alt => event.modifiers.command,
-            MultiCursorModifier::Cmd => event.modifiers.alt,
+            MultiCursorModifier::Alt => event.modifiers.alt,
+            MultiCursorModifier::Cmd => event.modifiers.command,
+            MultiCursorModifier::Ctrl => event.modifiers.control,
```

Comment:
I think these variables were poorly named (Sorry for merging that without reading it clearly!). 

Looking at https://github.com/zed-industries/zed/commit/71aaefc5b2413c3742601fc03501de93d04b1ad7, the intent was to make it such that "if the multi cursor modifier is alt, then cmd is used for opening links; if the multi cursor modifier is cmd, then alt is used for opening links.

I think that if the mulitcursor modifier is set to `ctrl`, `cmd` should still open links.

(That doesn't work on linux, where I think we need the default for clicking links to be `ctrl` but if you set the multi cursor modifier to `ctrl`, `alt` should open links – but I don't think this feature works on linux yet, so happy to punt that scope creep to a second PR).

Reply:
Ah I see what you mean, I guess what we could do is change how the enum is structured based on what platform you are, on Windows and Linux the options would be `Alt` and `Ctrl` and on MacOS the options would be `Alt` and `Cmd`. Implementing this shouldn't be too much of an issue anyways as `Alt` is the default key.

Let me know what you think of this idea, I could then either discard this PR and make a new one or change it up in this PR.

Reply:
The only thing that is not clear yet then is how we would handle the multi cursor modifier being alt, what will be used for opening links?

Reply:
Cmd on Mac, alt on windows/linux

Reply:
Hey, sorry for being so late with my response, I was busy throughout the week.

On Code Editors for Linux and Windows its usual that `Ctrl` is used to open links, not `Alt` or `Super` / `Win` (taking VS Code and Jetbrains IDEs as reference here, as that is whats used the most on these two platforms). Wouldn't it be better if the setting is set to `Ctrl` that this key is actually used to open links?

The changes that you wanted me to make are basically finished, I just want to make sure that we actually want `Alt` to be used to open links on Windows and Linux, not `Ctrl`.